src/SeleniumLibrary/base/context.py

```diff
@@ -60,6 +61,15 @@ def find_element(self, locator, tag=None, required=True, parent=None):
     def find_elements(self, locator, tag=None, parent=None):
         """Find all elements matching `locator`.
 
+        :param locator: Locator to use when searching the element.
+            See library documentation for the supported locator syntax.
+        :param tag: Limit searching only to these elements.
+        :param parent: Optional parent `WebElememt` to search child elements
+            from. By default search starts from the root using `WebDriver`.
+        :type parent: selenium.webdriver.remote.webelement.WebElement
+        :return: list of found `WebElement` or `[]` if element not found.
```

Comment:
Also this explanation can be enhanced and the paragraph below containing duplicate information removed.